### DISCUSSION TITLE
Different Colors and Intensities for different parts of Explorer

### DISCUSSION THREAD

**Discussion by Skyyblaze** [Maplespe/DWMBlurGlass#273] [Ideas]
I just saw this screenshot of Linux:

![grafik](https://github.com/Maplespe/DWMBlurGlass/assets/22203339/2095cd92-ccba-44a2-8423-ca2d44a92ffb)

and it gave me an idea for DWMBlurGlass. Would it be possible to have different parts of Explorer have different colors and maybe even blur intensities? Like in that screenshot the navigation-panel is darker than the rest of the window. Would that be something that would be possible and if yes, something that would be interesting for DWMBlurGlass?

**Comment by Maplespe** [comment]
DWMBlurGlass only works for DWM.
explorer is not in the scope of this project.
You can use ExplorerBlurMica and with some third party themes you should be able to achieve a similar effect.

**Reply by Skyyblaze** [reply]
Alright I understand, thanks for the clarification!
